docs/references/kontena-yml.md

```diff
@@ -313,3 +354,39 @@ nginx:
     fluentd-tag: docker.{{.Name}}
 
 ```
+
+## Example kontena.yml
+```
+loadbalancer:
+  image: kontena/lb:latest
+  ports:
+    - 80:80
+app:
+  build: .  
+  image: registry.kontena.local/example-app:latest
+  instances: 2
+  privileged: true  
+  links:
+    - loadbalancer
+  environment:
+    - DB_URL=%{project}-db.kontena.local
+    - KONTENA_LB_INTERNAL_PORT=80
+    - KONTENA_LB_VIRTUAL_HOSTS=www.my-app.com
+  deploy:
+    strategy: ha
+    wait_for_port: 80
+  hooks:
+    post_start:
+      - name: sleep
+        cmd: sleep 10
+        instances: *
+  log_driver: fluentd
```

Comment:
I think we should remove log_driver/log_opt because they are not something that user normally would use.